src/app/components/trees/trees-form/trees-form.component.ts

```diff
@@ -89,6 +115,18 @@ export class TreesFormComponent implements OnInit {
 
   async ngOnInit() {
 
+    this.getAllTreeTypes().then((resp) => {
+      this.treetypesList.push(...resp);
+    });
+
+    this.getFolderNodeTypes().then((resp) => {
+      this.folderNodeTypesList.push(...resp);
+    });
+
+    this.getFinalNodeTypes().then((resp) => {
```

Comment:
@albertobucesGdtl the change from "final" to "leaf" in the "dev" branch is because "final" is not widely used in tree-likes structures for these kind of nodes. "leaf" is more frequent.